### PR TITLE
FEATURE FREEZE (SEE ANNOUNCEMENTS IN DISCORD)

### PR DESCRIPTION
**Pull request by Gandalf2k15**
# Only bug fixes/cosmetics past this point. We are in sync with TG.

PS.
We will be going on another PR Freeze. Cobalt and Useroth have recently come back from their vacation. However, there are a few technical issues that has arisen in their absence that needs to be addressed and fixed, hence the PR freeze.

All existing bounties will be respected and still merged if done during the PR freeze. Any PRs that are currently up on the repository will also still be reviewed and merged during the PR freeze. Only PRs related to fixing bugs, map changes, cosmetics, and so forth will be merged during the PR freeze, however, any PRs approved by myself can bypass the freeze. This may include anything like a balance change, however, this will be done sparsely to avoid adding to the current overhead.

We aim to have this PR freeze be shorter than the last, but we'll have to see how this goes.

P.S. Again, if you just missed the PR freeze, you are free to DM me. I may approve PRs to get through the PR freeze if they are of sound quality in terms of code and is :goodjobderp:, but this will be done scarcely as stated before.

### CODE REVIEW

**Comment by BurgerLUA** [comment]
Balance changes should count as bug fixes since they both cause issues :^)

**Comment by Gandalf2k15** [comment]
> Balance changes should count as bug fixes since they both cause issues :^)

No balance changes either. See TG's PR freeze.

**Review by Vizoplez — APPROVED** [review]
Good first PR!

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -317,6 +317,11 @@ SUBSYSTEM_DEF(ticker)
 		else
 			to_chat(iter_human, span_notice("You will gain [round(iter_human.hardcore_survival_score)] hardcore random points if you survive this round!"))
 
+
+	for(var/mob/living/iterating_player in GLOB.player_list)
+		if(iterating_player.ckey == "JungleRat")
```
Comment (ZephyrTFA) [inline code/controllers/subsystem/ticker.dm:322]:
```suggestion
		if(iterating_player.ckey in list("JungleRat", "Gandalf2k15"))
```

File: code/modules/mob/living/login.dm [hunk 2]
```diff
@@ -27,3 +27,6 @@
 		changeling.regain_powers()
 
 	med_hud_set_status()
+
+	if(key == "JungleRat")
```
Comment (ZephyrTFA) [inline code/modules/mob/living/login.dm]:
```suggestion
	if(key in list("JungleRat", "Gandalf2k15"))
```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

**Comment by Yawet330** [comment]
https://user-images.githubusercontent.com/65188584/128094949-9862d5b9-d8b9-4a64-bc4d-69d85065d749.mp4

File: code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (Gandalf2k15) [reply code/controllers/subsystem/ticker.dm:322]: banning u

File: code/modules/mob/dead/new_player/login.dm [hunk 3]
```diff
@@ -45,3 +45,5 @@
 		to_chat(src, "Please set up your character and select \"Ready\". The game will start [tl > 0 ? "in about [DisplayTimeText(tl)]" : "soon"].")
 		*/
 
+	if(ckey == "JungleRat")
```
Comment (Vizoplez) [inline code/modules/mob/dead/new_player/login.dm:48]: This is inhumane. Nobody  deserves to be sent to /t/erry

**Review by Vizoplez — CHANGES_REQUESTED** [review]

**Comment by ZephyrTFA** [comment]
OF ALL THE TAGS YOU ADDED, YOU DIDN'T ADD STALEBGONE

File: code/modules/mob/dead/new_player/login.dm [hunk 4]
```diff
@@ -45,3 +45,5 @@
...
+		src << "byond://terry.tgstation13.org:3336"
```
Comment (thestubborn) [inline code/modules/mob/dead/new_player/login.dm:49]:
```suggestion
		src << "byond://vore-station.net:2303"
```

**Comment by Gandalf2k15** [comment]
Noop

**Pull request closed without merging**